src/classes/STG_PanelPaymentMapping_CTRL.cls

```diff
@@ -35,6 +35,7 @@
 */
 public with sharing class STG_PanelPaymentMapping_CTRL extends STG_Panel {
 
+    private static final String OPPORTUNITY_PAYMENT_OBJECT_NAME = 'npe01__OppPayment__c';
```

Comment:
Let's rename the constant to `PAYMENT_OBJECT_NAME` since explicit mention of the `Opportunity` is not required.

Reply:
Also, rather than hardcode the object api name, you can retrieve this dynamically using `Schema.SObjectType.npe01__OppPayment__c.getName()`.